src/System.Windows.Forms/tests/UnitTests/System/Windows/Forms/DataGridView.DataGridViewControlCollectionTests.cs

```diff
@@ -0,0 +1,23 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace System.Windows.Forms.Tests;
+
+public class DataGridView_DataGridViewControlCollectionTests
+{
+    [WinFormsFact]
+    public void DataGridViewControlCollection_Ctor_Control()
+    {
+        using var owner = new DataGridView();
```

Comment:
Could you change the objects we are instaniating in tests to have type on left hand side? e.g.
```suggestion
        using DataGridView owner = new();
```
Otherwise LGTM, thanks for adding tests! 👍